app/__main__.py

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env python
 
 import argparse
+import os
```

Comment:
Order at last. 😘

Reply:
I started using `isort` in Vim. It is life-changing.